lib/src/models/submission_impl.dart

```diff
@@ -237,6 +237,29 @@ class Submission extends SubmissionRef
     return previews;
   }
 
+  /// The variations of images for this [Submission] as a List of Map<String,SubmissionPreview>.
+  ///
+  ///  eg: [{"gif": [SubmissionPreview]}]
+  ///
+  /// Returns an empty List if the [Submission] does not have any image variations.
+  List<Map<String, SubmissionPreview>> get variants {
+    final previews = List<Map<String, SubmissionPreview>>();
```

Comment:
This is fine, but I'm surprised the analyzer didn't complain here. I was under the impression a lint was enabled to prefer `Map` and `List` instances to be created using `<Type1, Type2>{}` and `<Type>[]`, but maybe that's just enabled on the main Flutter repository... Anyway, not a big deal :)